libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -218,6 +220,24 @@ export interface ICSPDirective {
     };
 }
 
+/**
+ * @internal
+ */
+export interface ICustomApplicationSetting {
+    id: string;
+    type: string;
```

Comment:
what is `type`?

Reply:
I'm sorry @kandl, it is my mistake, this field should be removed from ICustomApplicationSetting